cfme/infrastructure/cluster.py

```diff
@@ -104,8 +87,21 @@ def is_displayed(self):
             self.entities.title.text == 'Clusters')
 
     toolbar = View.nested(ClusterToolbar)
-    entities = View.nested(ClusterEntities)
-    paginator = PaginationPane()
+    including_entities = View.include(BaseEntitiesView, use_parent=True)
+
+
+class ClusterAllFromProviderView(ClusterView):
+    """The all view page for clusters open from provider detail page"""
+    @property
+    def is_displayed(self):
+        """Determine if this page is currently being displayed"""
+        return (
+            self.logged_in_as_current_user and
+            self.entities.title.text == '{p}(All Clusters)'.format(p=self.context['object'].name))
```

Comment:
Shouldn't this be the provider name, not the cluster name? After clicking the clusters on the provider details page, I see the title:

"vSphere 6.5 (nested) (All Clusters)"

Reply:
actually, this view is used in only one place -  provider.get_clusters() and it obtains provider as a context['object']. However, it will have to be fixed if I turn it into nav destination

Reply:
Noted - I think having views defined in a module which assume the object for the view is a class from a different module is a tricky precedent to set.  We have other classes with 'AllFromProvider' destinations that are still registered against the local classes, and access the provider through the context object, they don't assume that the context object _is_ the provider.

Reply:
in fact,  this is not actual now because this view was moved under infra provider